src/mumble/LookConfig.h

```diff
@@ -54,6 +54,16 @@ class LookConfig : public ConfigWidget, Ui::LookConfig {
 		void load(const Settings &r) Q_DECL_OVERRIDE;
 		bool expert(bool) Q_DECL_OVERRIDE;
 		void themeDirectoryChanged();
+	protected:
+		void updateFontInfo(QLabel *l, const QFont &f);
+		void updateFontInfo();
+		void updateFontInfo(const Settings &r);
+		void changeFont(QFont & font);
+	protected slots:
+		void on_qpbFontLog_clicked();
+		void on_qpbFontInput_clicked();
+		void on_qpbFontTree_clicked();
+		void on_qpbFontAll_clicked();
```

Comment:
Missing documentation. A function header (Doxygen) explaining what these functions are intended for and what the parameters are for would be nice